parse/gosumdupes.sum

```diff
@@ -0,0 +1,24 @@
+github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7 h1:PqzgE6kAMi81xWQA2QIVxjWkFHptGgC547vchpUbtFo=
```

Comment:
maybe move this file into a `testdata` folder?

Reply:
In: 0502bd8